site/category.xml

```diff
@@ -26,10 +26,12 @@
 	<feature url="features/org.jboss.tools.openshift.test.feature_0.0.0.jar" id="org.jboss.tools.openshift.test.feature" version="0.0.0">
 		<category name="JBoss Tools OpenShift Nightly Build Update Site" />
 	</feature>
-
-        <feature url="features/org.jboss.tools.openshift.cdk.feature_0.0.0.jar" id="org.jboss.tools.openshift.cdk.feature" version="0.0.0">
-                <category name="JBoss Tools OpenShift Nightly Build Update Site" />
-        </feature>
+	<feature url="features/org.jboss.tools.openshift.cdk.feature_0.0.0.jar" id="org.jboss.tools.openshift.cdk.feature" version="0.0.0">
+		<category name="JBoss Tools OpenShift Nightly Build Update Site" />
+	</feature>
+	<feature url="features/org.jboss.tools.openshift.js.feature_0.0.0.jar" id="org.jboss.tools.openshift.js.feature" version="0.0.0">
```

Comment:
We also need to include this feature to devstudio installer and in the same categories we have openshift in JBossTools. cc: @nickboldt